idaes/models_extra/power_generation/costing/costing_dictionaries.py

```diff
@@ -66,3 +66,42 @@
 
 with open(os.path.join(directory, "sCO2_costing_parameters.json"), "r") as file:
     sCO2_costing_params = json.load(file)
+
+if not os.path.exists(
```

Comment:
I like that the exponents and other parameters are being combined into one dictionary. Does it make sense to just make the new dictionary offline and add it to the PR? That way we could remove the two json files being zipped together and the zipping code to keep the codebase cleaner.

Reply:
That will be a different PR (for backwards compatibility).